{
  "Faults": [
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-07T20:00:00.000+0000",
      "latitude": 51.539999872569815,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5057'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5057",
      "reference": "KT5057",
      "maxLongitude": -0.752,
      "etaUpdate": true,
      "minLatitude": 51.5341,
      "name": "SL6 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL6 6NJ",
        "SL6 6NN",
        "SL6 6NS",
        "SL6 6PR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7623,
              51.5451
            ],
            [
              -0.752,
              51.5451
            ],
            [
              -0.752,
              51.5341
            ],
            [
              -0.7623,
              51.5341
            ],
            [
              -0.7623,
              51.5451
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "96785bf1-08ac-4593-b43d-b103ded5a511",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.7623,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5451,
      "etrUpdate": true,
      "longitude": -0.7580547365465843,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-07T21:41:00.000+0000",
      "latitude": 50.754326244531654,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5083'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5083",
      "reference": "KT5083",
      "etaUpdate": true,
      "name": "BH23 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH23 7NA",
        "BH23 7NB"
      ],
      "location": {
        "coordinates": [
          -1.7728620998211704,
          50.754326244531654
        ],
        "type": "Point"
      },
      "UUID": "42e05fa6-5026-434e-94b7-c170b29c1427",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -1.7728620998211704,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T07:50:00.000+0000",
      "latitude": 51.600558617954086,
      "estimatedRestoration": "2023-04-08T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5158'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5158",
      "reference": "KT5158",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-08T07:05:00.000+0000",
      "affectedCustomerCount": 24,
      "name": "UB9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB9 6BZ",
        "UB9 6DD",
        "UB9 6DJ",
        "UB9 6DL",
        "UB9 6DQ"
      ],
      "location": {
        "coordinates": [
          -0.484634535841808,
          51.600558617954086
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2023-04-08T07:00:00.000+0000",
      "UUID": "830affc3-7f3c-4a92-a2cd-8fb0c49d8739",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.484634535841808,
      "resolved": false
    },
    {
      "latitude": 51.53088318844891,
      "type": "LV",
      "reference": "KT5228",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-08T09:35:00.000+0000",
      "affectedCustomerCount": 19,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "8513bfb9-8aad-4b69-b1d2-40411b3aefaa",
      "minLongitude": -0.7352,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7329294451128823,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T08:29:00.000+0000",
      "estimatedRestoration": "2023-04-08T20:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5228'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5228",
      "maxLongitude": -0.7285,
      "minLatitude": 51.5268,
      "name": "SL6 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL6 7LZ",
        "SL6 7NP",
        "SL6 7NU",
        "SL6 7NZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7352,
              51.5324
            ],
            [
              -0.7285,
              51.5324
            ],
            [
              -0.7285,
              51.5268
            ],
            [
              -0.7352,
              51.5268
            ],
            [
              -0.7352,
              51.5324
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-08T09:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5324,
      "etrUpdate": true
    },
    {
      "latitude": 51.62676421296015,
      "type": "LV",
      "reference": "KT5254",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-08T10:30:00.000+0000",
      "affectedCustomerCount": 18,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "501f788f-712b-48cc-b5a4-44fdb284b3eb",
      "minLongitude": -0.7451,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7405468248393907,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T09:18:00.000+0000",
      "estimatedRestoration": "2023-04-08T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5254'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5254",
      "maxLongitude": -0.7366,
      "minLatitude": 51.6226,
      "name": "HP11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "HP11 1BJ",
        "HP11 1BL",
        "HP11 1BN",
        "HP11 1BP",
        "HP11 1BZ",
        "HP11 1WQ",
        "HP13 6AA",
        "HP13 6AG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7451,
              51.6277
            ],
            [
              -0.7366,
              51.6277
            ],
            [
              -0.7366,
              51.6226
            ],
            [
              -0.7451,
              51.6226
            ],
            [
              -0.7451,
              51.6277
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-08T10:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.6277,
      "etrUpdate": true
    },
    {
      "latitude": 50.79382996229397,
      "type": "LV",
      "reference": "KT5305",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-08T11:55:00.000+0000",
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "82ae337a-692a-4e18-84f6-5d8b5d3b56d0",
      "minLongitude": -1.0933,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.092264374479854,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T10:40:00.000+0000",
      "estimatedRestoration": "2023-04-08T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5305'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5305",
      "maxLongitude": -1.0912,
      "minLatitude": 50.7932,
      "name": "PO5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO5 4AR",
        "PO5 4BP",
        "PO5 4BX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0933,
              50.7953
            ],
            [
              -1.0912,
              50.7953
            ],
            [
              -1.0912,
              50.7932
            ],
            [
              -1.0933,
              50.7932
            ],
            [
              -1.0933,
              50.7953
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-08T12:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7953,
      "etrUpdate": true
    },
    {
      "latitude": 51.23541940541268,
      "type": "HV",
      "reference": "KT5318",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a4ba0e6c-eca7-4c92-8c7c-00b798f44370",
      "minLongitude": -1.2918,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.289621691720462,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T11:06:00.000+0000",
      "estimatedRestoration": "2023-04-08T14:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5318'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5318",
      "maxLongitude": -1.2826,
      "minLatitude": 51.2304,
      "name": "RG28 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG28 7NE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2918,
              51.2386
            ],
            [
              -1.2826,
              51.2386
            ],
            [
              -1.2826,
              51.2304
            ],
            [
              -1.2918,
              51.2304
            ],
            [
              -1.2918,
              51.2386
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-08T11:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2386,
      "etrUpdate": true
    },
    {
      "latitude": 51.18639301398724,
      "type": "LV",
      "reference": "KT5321",
      "etaUpdate": true,
      "affectedCustomerCount": 18,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "c836ead6-4a37-457b-bf41-2cab4da3db3f",
      "minLongitude": -1.5126,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4871437761162087,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T11:14:00.000+0000",
      "estimatedRestoration": "2023-04-08T16:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5321'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5321",
      "maxLongitude": -1.4781,
      "minLatitude": 51.1686,
      "name": "SP11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SP11 0JF",
        "SP11 7RH",
        "SP11 7RJ",
        "SP11 7RL",
        "SP11 7RN",
        "SP11 7RS",
        "SP11 7RX",
        "SP11 7RZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5126,
              51.2762
            ],
            [
              -1.4781,
              51.2762
            ],
            [
              -1.4781,
              51.1686
            ],
            [
              -1.5126,
              51.1686
            ],
            [
              -1.5126,
              51.2762
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-08T13:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2762,
      "etrUpdate": true
    },
    {
      "latitude": 51.46704014643047,
      "type": "LV",
      "reference": "KT5360",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-08T13:10:00.000+0000",
      "affectedCustomerCount": 38,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "7dd70645-e70d-406d-9553-1e0982ed5ed3",
      "minLongitude": -2.1189,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.1149162125371066,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T12:42:00.000+0000",
      "estimatedRestoration": "2023-04-08T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5360'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5360",
      "maxLongitude": -2.1103,
      "minLatitude": 51.4636,
      "name": "SN15 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN15 1AA",
        "SN15 1AD",
        "SN15 1AZ",
        "SN15 1BL",
        "SN15 1BY",
        "SN15 1BZ",
        "SN15 1DA",
        "SN15 1DB",
        "SN15 1DG",
        "SN15 1DQ",
        "SN15 1DS",
        "SN15 1EB",
        "SN15 1QQ",
        "SN15 1TG",
        "SN15 1UB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1189,
              51.4706
            ],
            [
              -2.1103,
              51.4706
            ],
            [
              -2.1103,
              51.4636
            ],
            [
              -2.1189,
              51.4636
            ],
            [
              -2.1189,
              51.4706
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4706,
      "etrUpdate": true
    },
    {
      "latitude": 50.98745840157005,
      "type": "LV",
      "reference": "KT5374",
      "etaUpdate": true,
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b7e4767e-2923-47fd-9069-225bb9b06356",
      "minLongitude": -1.3625,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.359475773638782,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T13:21:00.000+0000",
      "estimatedRestoration": "2023-04-08T16:25:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5374'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5374",
      "maxLongitude": -1.3571,
      "minLatitude": 50.9865,
      "name": "SO50 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO50 4RL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3625,
              50.989
            ],
            [
              -1.3571,
              50.989
            ],
            [
              -1.3571,
              50.9865
            ],
            [
              -1.3625,
              50.9865
            ],
            [
              -1.3625,
              50.989
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.989,
      "etrUpdate": true
    },
    {
      "latitude": 56.99292885703944,
      "type": "PSI",
      "reference": "215242",
      "etaUpdate": true,
      "affectedCustomerCount": 14,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "9742bc4d-cfb2-444a-8fef-4e824278350e",
      "minLongitude": -7.5284,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -7.503337321433627,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T08:00:00.000+0000",
      "estimatedRestoration": "2023-04-08T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '215242'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/215242",
      "maxLongitude": -7.4488,
      "minLatitude": 56.9724,
      "name": "HS9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "HS9 5XS",
        "HS9 5XT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -7.5284,
              57.0087
            ],
            [
              -7.4488,
              57.0087
            ],
            [
              -7.4488,
              56.9724
            ],
            [
              -7.5284,
              56.9724
            ],
            [
              -7.5284,
              57.0087
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.0087,
      "etrUpdate": true
    },
    {
      "latitude": 57.41334848282476,
      "type": "HV",
      "reference": "KT5341",
      "etaUpdate": true,
      "affectedCustomerCount": 34,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "3dd42107-c89c-4506-8f62-a229c33c5d25",
      "minLongitude": -4.1727,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.138702801156019,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-08T12:26:00.000+0000",
      "estimatedRestoration": "2023-04-08T15:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT5341'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT5341",
      "maxLongitude": -4.0156,
      "minLatitude": 57.366,
      "name": "IV2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "IV1 2XH",
        "IV1 2XQ",
        "IV13 7YF",
        "IV2 5XL",
        "IV2 5XQ",
        "IV2 6WG",
        "IV2 6XN"
      ],
      "location": {
        "coordinates": [
          [
            [
              -4.1727,
              57.4489
            ],
            [
              -4.0156,
              57.4489
            ],
            [
              -4.0156,
              57.366
            ],
            [
              -4.1727,
              57.366
            ],
            [
              -4.1727,
              57.4489
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-08T13:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.4489,
      "etrUpdate": true
    }
  ]
}
